### DISCUSSION TITLE
CUT HERE FOR EXCEPTION DECODER

### DISCUSSION THREAD

**Discussion by petsch9** [revoxhere/duino-coin#1090] [Q&A]
Hi,
I have build first ESP8266 to connect to the server, but it stop by error, any suggestions?

Connected to WiFi!
Local IP address: ***
Fetched pool:   0

Connecting to Duino-Coin server...

--------------- CUT HERE FOR EXCEPTION DECODER ---------------

Soft WDT reset

>>>stack>>>

ctx: cont
sp: 3ffffd00 end: 3fffffc0 offset: 01a0
3ffffea0:  3fffdad0 3ffeea10 3ffee8e4 402022fc  
3ffffeb0:  0011001f 00000000 00000000 feefeffe  
3ffffec0:  00000000 00000000 00000000 00000000  
3ffffed0:  ffffffff 00000000 00000000 feef000a  
3ffffee0:  00000000 feefeffe 00000000 00000000  
3ffffef0:  00000000 00000000 0011001f 00000000  
3fffff00:  40201070 3fff0214 00000000 3fff0204  
3fffff10:  3fff01f4 feefef20 00000000 0011001f  
3fffff20:  00000000 00000000 0010001f 00000000  
3fffff30:  00000000 000f000f 00000000 00000000  
3fffff40:  000f000f 00000000 00000000 000e000f  
3fffff50:  00000000 00000000 feefeffe 00000000  
3fffff60:  00000000 feefeffe 00000000 00000000  
3fffff70:  0075007f 00000000 00000000 0075007f  
3fffff80:  00000000 feefeffe feefeffe feefeffe  
3fffff90:  feefeffe feefeffe feefeffe 3ffeeb54  
3fffffa0:  3fffdad0 00000000 3ffeeb40 40208ad8  
3fffffb0:  feefeffe feefeffe 3ffe85dc 40100e39  
<<<stack<<<

--------------- CUT HERE FOR EXCEPTION DECODER ---------------

**Comment by JK-Rolling** [comment]
is this esp01? esp01 instability usually caused by insufficient power

**Comment by revoxhere** [comment]
can you access https://server.duinocoin.com/getPool in your browser?

**Reply by petsch9** [reply]
nop.

{"error":"You're using a Proxy, VPN, VPS or a similar service. This is against our Terms of Service","success":false}

**Reply by petsch9** [reply]
I am not on a VPN or similar service, any ideas to solve this?

**Reply by petsch9** [reply]
I think the project ends here for me. my internet service provider won't help

**Reply by petsch9** [reply]
@revoxhere I am now on another IP and now i get the next results i hope yoy have some more ideas

on url request in browser: 
{"name":"null-pool-1","ip":"5.230.70.190","port":6000,"success":true}

But my ESD returns the following:

Connected to WiFi!
Local IP address: blabla
[HTTP] GET... failed, error: connection failed

Connecting to Duino-Coin server...

--------------- CUT HERE FOR EXCEPTION DECODER ---------------

Soft WDT reset

**Reply by petsch9** [reply]
Sorry for spam, but i have it working now :)
